src/api.js

```diff
@@ -8,3 +8,9 @@ export const getReviews = () => {
     return data.reviews;
   });
 };
+
+export const getReviewById = (review_id) => {
+  return gamesAPI.get(`/reviews/${review_id}`).then(({ data }) => {
```

Comment:
Really nice destructuring here! 

You could take it one step further and use { data : {review} }! It's another layer of destructuring, once you have destructured 'data' then you have access to the keys on it with the above syntax.

This would mean you don't have to send data.review and just send review!